bootstrap/components/sources/kafka-source-topology-component.json

```diff
@@ -37,10 +37,11 @@
       {
         "uiName": "Reader schema version",
         "fieldName": "readerSchemaVersion",
-        "isOptional": true,
+        "isOptional": false,
```

Comment:
@satishd This change is kind of a backwards incompatible so its not possible to have a seamless experience. Moreover, I think UI forcing the user to pick a reader schema version in newer version is not a big inconvenience. I agree with you on showing an info message. However, setting the right reader version approach you suggested is not clear and seems like an overkill.  What is the right reader version here? You mentioned you get that by sending current schema text and name and get it from SR. Where do we get schema text from? And which version? By default in earlier versions if reader schema version is not specified the writer schema version from payload is used to deserialize.